libfwupd/fwupd-enums.h

```diff
@@ -1101,6 +1101,44 @@ typedef enum {
 	FWUPD_VERSION_FORMAT_LAST
 } FwupdVersionFormat;
 
+/**
+ * FWUPD_VERSION_FORMAT_AABB_CCDD:
+ *
+ * A "pair" version format where a word is split into two sections
+ */
+#define FWUPD_VERSION_FORMAT_AABB_CCDD FWUPD_VERSION_FORMAT_PAIR
```

Comment:
my worry with this too is that for the LVFS we make the vendors specify what the version format is in the metainfo file. Would they specify `LVFS::VersionFormat=="AABB_CCDD"` or `LVFS::VersionFormat=="aabb.ccdd"` etc?